### PR TITLE
Change TimeGeneration configuration script

### PR DESCRIPTION
**Pull request by NoamLandress**
A new script that changes the TimeGenerated configuration to point to log creation time instead of log collection time.

### CODE REVIEW

File: DataConnectors/CEF/TimeGenerated.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+def change_events_timegenerated():
+    """
+    :return: True if successfully changed the TimeGenerated configuration
+    """
+    grep = subprocess.Popen(["grep", "-i", "'Timestamp' => OMS::Common::fast_utc_to_iso8601_format(Time.now.utc),",
+                             oms_agent_field_mapping_configuration], stdout=subprocess.PIPE)
+    o, e = grep.communicate()
+    if not o:
```
Comment (ofshomro) [inline DataConnectors/CEF/TimeGenerated.py:27]: Also if e?

Comment (NoamLandress) [reply DataConnectors/CEF/TimeGenerated.py]: Fixed it.

**Comment by morshabi** [comment]
@NoamLandress is PR still relevant?

**Comment by NoamLandress** [comment]
Script Was merged in a different PR- https://github.com/Azure/Azure-Sentinel/pull/1288

**Pull request closed without merging**